tool/builder/bin/main.dart

```diff
@@ -0,0 +1,14 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
As this is a new file the year should be 2023. Unless it's copied from somewhere?

Reply:
This is the standard Flutter authors license.